<launch>
  <arg name="positions_file" default="/home/roomac/roomac_data/positions.yaml" />
  <arg name="simulation" default="false" />

  <!-- how long to wait for stabilization of artag position before picking is started, has to be longer for simulation -->
  <arg if="$(arg simulation)" name="wait_time_before_picking_action" value="5.0" />
  <arg unless="$(arg simulation)" name="wait_time_before_picking_action" value="0.1" />

  <node name="pick_and_bring_manager" pkg="roomac" type="pick_and_bring_manager_node.py" output="screen">
    <rosparam command="load" file="$(find roomac)/config/pick_and_bring_manager_config.yaml" subst_value="true" />
  </node>
</launch>